packages/common/src/hooks/useDebounce/useDebouncedValue.stories.tsx

```diff
@@ -26,18 +27,12 @@ const Value: Story = () => {
         />
       </Box>
       <Box gap={2} mt={2} flexDirection="row" display="flex">
-        <Button
-          variant="contained"
-          onClick={() => setNumber(state => (state += 1))}
-        >
+        <BaseButton onClick={() => setNumber(state => (state += 1))}>
           Increment
-        </Button>
-        <Button
-          variant="contained"
-          onClick={() => setNumber(state => (state -= 1))}
-        >
+        </BaseButton>
+        <BaseButton onClick={() => setNumber(state => (state -= 1))}>
           Decrement
-        </Button>
+        </BaseButton>
```

Comment:
Can just use our `BaseButton` now and it's styled as the contained, primary variant (maybe the outlined variant is better 🤷 )